model/sparql/GenericSparql.php

```diff
@@ -1402,6 +1402,27 @@ private function generateLabelQuery($uri, $lang) {
         return $query;
     }
 
+
+    /**
+     * @param string $uri
+     * @param string $lang
+     * @return string sparql query string
+     */
+    private function generateOtherLabelQuery($uri, $lang) {
+        $fcl = $this->generateFromClause();
+        $labelcondLabel = ($lang) ? "FILTER( langMatches(lang(?val), '$lang') )" : "";
+        $query = <<<EOQ
+CONSTRUCT { <$uri> ?prop ?val . } $fcl
```

Comment:
It's not wrong to use a CONSTRUCT here, but the implied RDF serialization (Fuseki) and parsing (EasyRdf) seems like unnecessary overhead. This could easily be changed into `SELECT ?prop ?val` instead.

Reply:
Can do. This was for my own convenience since listing the literals from the response graph seemed the simplest. I'll change it to an iterable result.